src/components/dialogs/create-case-dialog.js

```diff
@@ -85,7 +85,15 @@ export function CreateCaseDialog({ onClose, open }) {
         })
             .then()
             .catch((message) => {
-                snackbarMessage(message, 'caseCreationError', { name });
+                message.trim().startsWith('422')
```

Comment:
You changed backend to throw a 422 http status, check this status instead of parsing the error message

Reply:
done